resources/shaders/dxr_pathtracer_functions.hlsl

```diff
@@ -34,6 +33,36 @@ float3 TraceColorRay(float3 origin, float3 direction, unsigned int depth, unsign
 	return payload.color;
 }
 
+float3 TraceColorRayCone(float3 origin, float3 direction, unsigned int depth, unsigned int seed, RayCone cone)
+{
+	if (depth >= MAX_RECURSION)
+	{
+		return float3(0, 0, 0);
+	}
+
+	// Define a ray, consisting of origin, direction, and the min-max distance values
+	RayDesc ray;
+	ray.Origin = origin;
+	ray.Direction = direction;
+	ray.TMin = 0;
+	ray.TMax = 1.0e38f;
```

Comment:
Isn't there a way to get this value via HLSL of something like that? Just curious... I don't like "magic numbers", hence I'm asking.

Reply:
not new code.

Reply:
So it's not worth resolving? If not, feel free to commit the file the way it is. :)